pages/inspector/start-using-inspector.mdx

```diff
@@ -29,32 +29,7 @@ Inspector does not process property values or PII data, only event names, proper
 
 TL;DR: No PII data ever flows through Avo servers; only event schemas.
 
-## Learning about Inspector issues
-
-After [setting up your sources](/inspector/inspector-overview#configuring-sources) to send data to the Inspector with the direct CDP integrations or [Avo Inspector SDKs](/implementation/avo-inspector-sdk-reference) you will see the data appearing in the the Inspector tab.
-There are two Inspector overviews available for your incoming data and issues, the [Issues view](/inspector/inspector-issues-view) and the [Events view](/inspector/inspector-events-view) – and you can set up [Slack Alerts](/inspector/inspector-slack-alerts) to be notified about new issues.
-
-### Issues view
-
-The Inspector issues view displays information about anomalies and discrepancies seen in your production data for the past 24 hours. This view allows you to sort and filter issues to uncover those relevant to you and gives you the overview and information needed to prioritize which of them to act on.
-
-![Image shows an example default view of the Inspector issues view.](/images/inspector/inspector-issues-default.png)
-
-Learn more about the [Issues view here](/inspector/inspector-issues-view).
-
-### Events view
-
-The Inspector events view provides a report on all event tracking Inspector has seen in your development, staging and production environments. You can use the dashboard to evaluate the health of your tracking implementation and prioritize issues.
-
-![Inspector tab](/images/workspace/nav-inspector.png)
-
-Learn more about the [Events view here](/inspector/inspector-events-view).
-
-> The production environment in Inspector is designed to handle large amounts of data and has up to 2 hours delay from receiving events to displaying them on a dashboard.
-
-> Events on dev and staging have no delay in processing and will be shown on the dashboard immediately.
-
-## How to send metadata to the Inspector?
```

Comment:
I'd like to keep the product overview above the setup instructions. You don't have enough motivation to consider setups until you've learned more about the end product